### ISSUE TITLE
I18n::InvalidLocale if imagemagick/graphicsmagick is not installed

### ISSUE THREAD

**Issue by gtournie** [carrierwaveuploader/carrierwave#1729]
Hey,

I got this error when using carrierwave without imagemagick / graphicsmagick installed:
`I18n::InvalidLocale (:en is not a valid locale)`

I had to add :en in my locales to find out what the problem was:

```
Image Failed to manipulate with MiniMagick, maybe it is not an image?
Original Error: ImageMagick/GraphicsMagick is not installed
```

Here was my application.rb:

```
config.i18n.available_locales = [:fr]
config.i18n.default_locale = :fr
```

and my locale file:

```
fr:
  errors:
    messages:
      carrierwave_processing_error: "Impossible de redimensionner l'image."
      carrierwave_integrity_error: "Ce n'est pas une image."
      carrierwave_download_error: "Impossible de télécharger l'image."
      extension_white_list_error: "Vous n'êtes pas autorisé à uploader des fichiers %{extension}, types autorisés: %{allowed_types}"
      extension_black_list_error: "Vous n'êtes pas autorisé à uploader des fichiers %{extension}, types interdits: %{prohibited_types}"
      mime_type_pattern_white_list_error: "Vous n'êtes pas autorisé à uploader des fichiers de type %{content_type}"
      mime_type_pattern_black_list_error: "Vous n'êtes pas autorisé à uploader des fichiers de type %{content_type}"
      rmagick_processing_error: "La manipulation d'image avec rmagick a échoué. Peut-être que ce n'est pas une image ? Erreur originale: %{e}"
      mime_types_processing_error: "Le traitement de fichier avec MIME::Types a échoué. Peut-être que ce n'est pas un type valide ? Erreur originale: %{e}"
      mini_magick_processing_error: "La manipulation d'image avec MiniMagick a échoué. Peut-être que ce n'est pas une image ? Erreur originale: %{e}"
      min_size_error: "La taille du fichier doit être supérieure à %{min_size}"
      max_size_error: "La taille du fichier doit être inférieure à %{max_size}"
```

Why it doesn't use the :fr locale ?

**Comment by thomasfedb** [comment]
@mehlah Wondered if you might have thoughts on this?

**Comment by mshibuya** [comment]
Fixed by #2083.
